cmd/gateway-main.go

```diff
@@ -191,6 +205,8 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 
 	// Create new notification system.
 	globalNotificationSys = NewNotificationSys(globalServerConfig, EndpointList{})
+	handleNotifyEnvVar(&globalNotificationSys.targetList)
```

Comment:
Instead of passing a pointer (looks scary), you could have value filled in `globalServerConfig`.  If this is not possible, you could pass `targetList` as arg to `NewNotificationSsys` than `globalServerConfig`